addons/account/models/account_move.py

```diff
@@ -3350,9 +3374,6 @@ def _compute_tax_audit(self):
             audit_str = ''
             for tag in record.tag_ids:
 
-                caba_origin_inv_type = record.move_id.type
-                caba_origin_inv_journal_type = record.journal_id.type
-
```

Comment:
Put this in another commit, so that we don't mix it with the content of the fix.